lang/cs/Org.Apache.REEF.Experimental/Org.Apache.REEF.Experimental.csproj

```diff
@@ -0,0 +1,68 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Please align these .csproj files with the other projects. For instance, we always include some common props files and set the output directory to a shared place for the whole solution.